aws/service.go

```diff
@@ -861,7 +861,7 @@ func GuardDutyClient(ctx context.Context, d *plugin.QueryData) (*guardduty.Clien
 }
 
 func HealthClient(ctx context.Context, d *plugin.QueryData) (*health.Client, error) {
-	cfg, err := getClientForQueryRegion(ctx, d)
+	cfg, err := getClientForDefaultRegion(ctx, d)
```

Comment:
I'm still [encountering this issue](https://github.com/turbot/steampipe-plugin-aws/issues/2449#issuecomment-2797326412) with the latest version of the plugin.

Is the default region the right choice here?

Shouldn't it be only one of `us-east-1` or `us-east-2` for main AWS partition,  or `us-gov-west-1` for GovCloud?

https://docs.aws.amazon.com/general/latest/gr/awshealth.html

Reply:
Submitted https://github.com/turbot/steampipe-plugin-aws/pull/2470 to fix that.